_inc/client/appearance/index.jsx

```diff
@@ -90,29 +90,32 @@ export const Page = ( props ) => {
 		);
 	} );
 
+	let holidaySnowCard = Initial_State.showHolidaySnow ? (
```

Comment:
This is not ok because it doesn't check the current app state but the initial one. You need to create a reducer for this in `state/settings/reducer.js`. Let me know if you need help, I can create one and integrate it if you want.